recipes/ffx/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.3.4" %}
+
+package:
+  name: ffx
+  version: {{ version }}
+
+source:
+    git_url: https://github.com/Ohjeah/ffx
+    git_tag: d1f027ed5f4a543c41af13f4b9a2be882ec4dd6a
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
The Linux docker image changed and some tools were removed. Can you add `git` to the build requirements? Or better yet. Change,

```yaml
source:
    git_url: https://github.com/Ohjeah/ffx
    git_tag: d1f027ed5f4a543c41af13f4b9a2be882ec4dd6a
```

to (the first line at the top right below the version)

```yaml
{% set commit = "d1f027ed5f4a543c41af13f4b9a2be882ec4dd6a" %}

source:
    fn: {{ commit }}.tar.gz
    url: https://github.com/Ohjeah/ffx/archive/{{ commit }}.tar.gz
    sha256: 44977347d9c9a60838283e3a684efe10a7772129c7d604fdc6695e85b9d7fc1f

```